node/README.md

```diff
@@ -0,0 +1,18 @@
+# Node
+
+ℹ️ A node -  in Polkadot - is a binary executable, whose primary purpose is to execute the [runtime](../runtime/README.md).
+
+🔗 It communicates with other nodes in the network, and aims for
+[consensus](https://wiki.polkadot.network/docs/learn-consensus) among them.
+
+⚙️ It acts as a remote procedure call (RPC) server, allowing interaction with the blockchain.
+
+👉 Learn more about the architecture, and the difference between a node and a runtime
+[here](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/wasm_meta_protocol/index.html).
+
+👇 Here are the most important files in this node template:
```

Comment:
I wouldn't try to provide a exhaustive list of the most interesting components of the node.
But I would point that the node is customizable too, like the runtime. And that some direct entry points for that customization are the ones in the list.